glpi-project/glpi
# Test update using utf8mb4 DB

| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -

1) Add a test to ensure DB migration from 9.5.3 (latest version not supporting utf8mb4) will not produce warnings if DB il configured using the `--use-utf8mb4` flag.
This test should not pass as `install/update_95_xx/recurrentchange.php` as to be fixed.

2) Prevent the `--force` param to replay twice the last migration.

3) Enhance test suite log:
- Display ansi colors on console commands
- Echo comments in update tests
- Use tee + grep to preserve log output
- Switch to en_GB for default lang
